src/Page/Guide/View.elm

```diff
@@ -89,7 +89,7 @@ viewImageColumn language guide =
                 text ""
         ]
     , div [ css [ hideFromPrint ] ] [ text (t InCategory ++ t (categorySlugToKey guide.categorySlug)) ]
-    , div [] (markdownToHtml guide.fullTextMarkdown)
+    , div [] ([ h2 [] [ viewHeaderIcon "../../../public/images/icons/NextDoorNature_icons-text.svg", text (t GuideTextHeader) ] ] ++ markdownToHtml guide.fullTextMarkdown)
```

Comment:
I feel like this string needs moving to the copy file - or at least into a var. the path should work as `/images/icons...`. It's confusing like this & I'm surprised it works since it's not nested as such in `dist` build

Reply:
done